src/test/java/nurseybook/logic/commands/AddTagCommandTest.java

```diff
@@ -1,33 +1,30 @@
-package seedu.address.logic.commands;
-
+package nurseybook.logic.commands;
+
+import static nurseybook.logic.commands.CommandTestUtil.SET_ONE_TAG;
+import static nurseybook.logic.commands.CommandTestUtil.SET_TWO_TAGS;
+import static nurseybook.logic.commands.CommandTestUtil.VALID_TAG_DIABETES;
+import static nurseybook.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
+import static nurseybook.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static nurseybook.testutil.TypicalIndexes.INDEX_FIRST;
+import static nurseybook.testutil.TypicalIndexes.INDEX_SECOND;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.commands.CommandTestUtil.SET_ONE_TAG;
-import static seedu.address.logic.commands.CommandTestUtil.SET_TWO_TAGS;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_DIABETES;
-import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.logic.commands.CommandTestUtil.showElderlyAtIndex;
-import static seedu.address.testutil.TypicalElderlies.getTypicalAddressBook;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST;
-import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND;
-import static seedu.address.testutil.TypicalIndexes.INDEX_THIRD;
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.commons.core.Messages;
-import seedu.address.commons.core.index.Index;
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.UserPrefs;
-import seedu.address.model.person.Elderly;
-import seedu.address.testutil.ElderlyBuilder;
-
+import nurseybook.commons.core.Messages;
+import nurseybook.commons.core.index.Index;
+import nurseybook.model.Model;
+import nurseybook.model.ModelManager;
+import nurseybook.model.NurseyBook;
+import nurseybook.model.UserPrefs;
+import nurseybook.model.person.Elderly;
+import nurseybook.testutil.ElderlyBuilder;
+import nurseybook.testutil.TypicalElderlies;
+import nurseybook.testutil.TypicalIndexes;
 public class AddTagCommandTest {
 
-    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model model = new ModelManager(TypicalElderlies.getTypicalNurseyBook(), new UserPrefs());
```

Comment:
Similarly, importing `TypicalElderlies` at the start of the file would be better?

Recurring in lines 48, 52.